mmedit/models/editors/deepfillv1/deepfill_disc.py

```diff
@@ -47,22 +46,11 @@ def forward(self, x):
 
         return global_pred, local_pred
 
-    def init_weights(self, pretrained=None):
-        """Init weights for models.
+    def init_weights(self):
+        """Init weights for models."""
```

Comment:
I think we should add the following code here:
```python
        if self.init_cfg is None and self.init_cfg['type'] == 'Pretrained':
            super().init_weights()
            return
```
Otherwise, this model can never load the pre-trained model.